std/conv.d

```diff
@@ -6136,6 +6140,88 @@ private auto hexStrLiteral(String)(scope String hexData)
     assert(hexString!"ab cd" == hexString!"ABCD");
 }
 
+/************************************************
+ * Convert string of `[0-9A-Fa-f]` characters to an array of ubytes.
+ *
+ * Other characters in the string are considered separators.
+ * Adjacent pairs of characters form the nibbles of the ubyte.
+ * Non-pairing characters form the low nibble of the ubyte.
+ * The returned array does not have a terminating zero appended.
+ * The GC is used because `hexData()` is intended to be used
+ * at compile time to, for example, initialize manifest constants
+ * and statically initialize global variables.
+ * Params:
+ *      s = string of data to convert
+ * Returns:
+ *      unique immutable array of ubytes allocated on the GC heap
+ * References:
+ *      $(LREF hexString)
+ */
+immutable(ubyte)[] hexData(const(char)[] s) pure nothrow @trusted
+{
+    ubyte[] result;
+    result.length = (s.length + 1) / 2;
+    size_t i;
+
+    void append(ubyte u)
+    {
+        result[i++] = u;
+    }
+
+    ubyte u;
+    uint digits;
+    foreach (char c; s)
+    {
+        if ('0' <= c && c <= '9')
+            c -= '0';
+        else if ('a' <= c && c <= 'f')
+            c -= 'a' - 10;
+        else if ('A' <= c && c <= 'F')
+            c -= 'A' - 10;
+        else
+        {
+            if (digits)
+                append(u);
+            u = 0;
+            digits = 0;
+            continue;
+        }
+
+
```

Comment:
One line space will suffice.

Reply:
I use two lines to separate one function from the next unrelated one.